mesh/mesh_readers.cpp

```diff
@@ -2735,6 +2868,26 @@ void Mesh::ReadGmshMesh(std::istream &input, int &curved, int &read_gf)
       }
    } // we reach the end of the file
 
+   // Process set names
```

Comment:
nitpick/optional:

maybe:
```
   if(!phys_names_by_dim.empty()) 
   {
      // Process boundary attribute set names
      for(auto const &attr: phys_names_by_dim[Dim-1])
      {
         AddToBdrAttributeSet(attr.second, attr.first);
      }

      // Process element attribute set names
      for(auto const &attr: phys_names_by_dim[Dim])
      {
         AddToAttributeSet(attr.second, attr.first);
      }
   }
```

Reply:
I missed this one. I'm glad you found it.